deployment/README.md

```diff
@@ -0,0 +1,74 @@
+# Amazon Web Services Deployment
+
+Amazon Web Services deployment is driven by [Terraform](https://terraform.io/), [Ansible](https://www.ansible.com/), and the [AWS Command Line Interface (CLI)](http://aws.amazon.com/cli/).
+
+## Table of Contents
+
+* [Deployment](#deployment)
+    * [Pre-Deployment Configuration](#pre-deployment-configuration)
+    * [SSH Keys](#ssh-keys)
+    * [DISTRICTBUILDER_SETTINGS_BUCKET](#districtbuilder_settings_bucket)
+    * [User Data](#user-data)
+    * [`scripts/infra`](#scriptsinfra)
+    * [Loading Shapefile data](#loading-shapefile-data)
+
+## Deployment
+
+### Pre Deployment Configuration
+
+#### SSH Keys
+
+You'll also need to generate an SSH Keypair using the AWS EC2 console. Download the private key, and store it at `~/.ssh/district-builder.pem`.
+
+#### DISTRICTBUILDER_SETTINGS_BUCKET
+
+Before running `scripts/infra`, create an AWS S3 bucket to house the terraform remote state and configuration file. Make note of the bucket name, it will be the value of the `DISTRICTBUILDER_SETTINGS_BUCKET` environment variable later on.
+
+You'll also need to create a `terraform.tfvars` file, which contains the configuration parameters for your infrastructure and application. You can see the available configuration options in [`sample.tfvars`](./terraform/sample.tfvars). Modify this file to suit your needs, and store it on S3 at `s3://${DISTRICTBUILDER_SETTINGS_BUCKET}/terraform/terraform.tfvars`. Note that Application server configuration such as admin usernames and passwords are prefied with `districtbuilder_` (i.e. `districtbuilder_admin_user`).
+
+#### User Data
+
+You can also upload your own DistrictBuilder `config.xml` (optional) and shapefile zip (required) to the App server as a part of provisioning. Simply add a DistrictBuilder `config.xml` file and `districtbuilder_data.zip` in the [`user-data`](./user-data/) folder, and the provisioner will upload them to the server for you. 
```

Comment:
We provide a config template (`config.dist.xml`), but I think we should still say `config.xml` is required